src/splinterdb.c

```diff
@@ -88,7 +88,7 @@ splinterdb_config_set_defaults(splinterdb_config *cfg)
       cfg->filter_index_size = 256;
    }
    if (!cfg->filter_remainder_size) {
-      cfg->filter_remainder_size = 6;
+      cfg->filter_remainder_size = 4;
```

Comment:
I haven't checked the logic here, but if this change is what will avoid printing the fingerprint array related message, then this change is fine.

Reply:
Yes, this was happening automatically in the config auditor, and now it's just the default.